.github/actions/submit-delete-k8s-job/action.yml

```diff
@@ -32,6 +33,37 @@ runs:
           
           # Stream logs
           kubectl logs --all-containers=true --all-pods=true --follow job/${{ inputs.job-name }}
-          
+
+          # Check whether the job succeeded or failed 
+          while readarray -d : -t status < <(kubectl get job/${{ inputs.job-name }} -o 'jsonpath={.status.failed}:{.status.succeeded}'); do
+            failures="${status[0]:-0}"
+            successes="${status[1]:-0}"
+            total=$((failures + successes))
+
+            if [[ $total -lt 2 ]]; then
```

Comment:
I believe this needs to match the parallelism configured in the job, which is not immediately known in this workflow.

Either it should be an input, or it should be extracted programmatically here.